src/components/ContextMenu.scss

```diff
@@ -73,4 +73,18 @@
   .context-menu-option:focus {
     z-index: 1;
   }
+
+  @media #{$media-query} {
+    .context-menu-option {
+      display: block;
+
+      .context-menu-option__label {
+        margin-inline-end: 0 !important;
```

Comment:
is it possible to have a solution without `!important`? We should discourage this rule.

Reply:
I have tried without `!important` but its not working!! guide me if you have any other solution 🤗

Reply:
I will try out something ...

Reply:
@harshilparmar can you also replace https://github.com/excalidraw/excalidraw/blob/bc414ccaaf731264b22c628f8f7c512cc9f2bff4/src/components/ContextMenu.scss#L50 with `context-menu-option__label` then we should be able to remove `!important`

Reply:
you can put the @media query inside the "normal" css block. Then no ``!important``is nesseccary.
```
    div:nth-child(1) {
      justify-self: start;
      margin-inline-end: 20px;

      @media #{$media-query} {
        margin-inline-end: 0;
      }
    }
```
and to avoid ``nth-child`` it can be changed to:
```
    .context-menu-option__label {
      justify-self: start;
      margin-inline-end: 20px;

      @media #{$media-query} {
        margin-inline-end: 0;
      }
    }
```